pkgs/tools/misc/journaldriver/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib, fetchFromGitHub, rustPlatform, pkgconfig, openssl, systemd }:
+
+rustPlatform.buildRustPackage rec {
+  name        = "journaldriver";
+  version     = "1.0.0-rc2";
+  cargoSha256 = "04llhriwsrjqnkbjgd22nhci6zmhadclnd8r2bw5092gwdamf49k";
+
+  src = fetchFromGitHub {
+    owner  = "aprilabank";
+    repo   = "journaldriver";
+    rev    = "v${version}";
+    sha256 = "19vb6hk65cvf4401hjg52q8ihkhcykrx72llv7m08bwpz59rycs2";
+  };
+
+  # Disable tests in nixpkgs
+  doCheck = false;
```

Comment:
What's the reason for the tests being disabled?

Reply:
Mostly decided to do that due to the time it takes to build the entire dependency chain, which will happen twice for running tests & non-tests.

The release versions have already had tests run (otherwise they wouldn't have gone into upstream master) and since all dependencies are pinned it should be fine.

Normally I'd package a Rust tool by using the `buildRustCrate` (which doesn't run tests at all), but that's not resolving the dependency on libsystemd correctly for some reason and has no obvious (read: documented) way to override the dependencies it figures out.

If you think tests should be enabled I don't mind enabling them :)

Reply:
As long as the tests work, I think they should be on by default. Hydra will have it in the cache anyways, so it doesn't really matter how long it takes for the end user. Usually the only good reason to disable tests is because the tests require network access or something else that Nix doesn't have in the sandbox.

Reply:
:+1: Updating with tests enabled.